File: api/HealthCare/Models/Pacijent.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace HealthCare.Models
{
    [Table("Pacijent")]
    public class Pacijent : Korisnik
    {
        public DateTime DatumRodenja { get; set; }
        public string MjestoRodenja { get; set; }

        public ZdravstvenaLegitimacija ZdravstvenaLegitimacija { get; set; }
        public Karton Karton { get; set; }
        public ICollection<Nalaz> Nalazi { get; set; }
        public ICollection<Uputnica> Uputnice { get; set; }
        public ICollection<Termin> Termini { get; set; }
        public ICollection<Recept> Recepti { get; set; }

    }
}

File: angular/src/app/interfaces/pacijent.ts
import { Karton } from "./karton";
import { Recept } from "./recept";
import { ZdravstvenaLegitimacija } from "./zdravstvenaLegitimacija";

export interface Pacijent {
    id: number;
    email: string;
    ime: string;
    prezime: string;
    brojTelefona: string;
    datumRodenja: string;
    mjestoRodenja: string;
    slika: any;
    zdravstvenaLegitimacija: ZdravstvenaLegitimacija;
    karton: Karton;
}
